pkg/manifest/anaconda_installer_iso_tree.go

```diff
@@ -17,16 +17,16 @@ import (
 	"github.com/osbuild/images/pkg/ostree"
 )
 
-type RootfsType uint64
+type ISORootfsType uint64
 
 // These constants are used by the ISO images to control the style of the root filesystem
 const ( // Rootfs type enum
-	SquashfsExt4Rootfs RootfsType = iota // Create an EXT4 rootfs compressed by Squashfs
-	SquashfsRootfs                       // Create a plain squashfs rootfs
-	ErofsRootfs                          // Create a plain erofs rootfs
+	SquashfsExt4Rootfs ISORootfsType = iota // Create an EXT4 rootfs compressed by Squashfs
+	SquashfsRootfs                          // Create a plain squashfs rootfs
+	ErofsRootfs                             // Create a plain erofs rootfs
```

Comment:
The enum values could be renamed here too for the same reason.  In other words, `SquashfsExt4ISORootfs` is a better name than `SquashfsExt4Rootfs` (ignoring the length).

It's fine to leave this for a follow-up though.